### ISSUE TITLE
Backport Sanity test to 1.x branch

### ISSUE THREAD

**Issue by DarshitChanpura** [opensearch-project/security#2398]
Backport sanity tests introduced in #1958.

This change isn't a simple backport as it involves modifying build.gradle as significant changes were made to it moving from 1.x to 2.x.

**Comment by scrawfor99** [comment]
[Triage] Thank you @DarshitChanpura, this seems like a good issue that will be addressed to allow for backporting to 1.3 as well.

**Comment by scrawfor99** [comment]
@DarshitChanpura, I can help pick this up.

**Comment by peternied** [comment]
I don't think we can 'backport' these tests.  I think we need to reauthor these tests in a way that will work in the 1.X ecosystem.  

Have we considered replacing `OpenSearchRestTestCase` and creating a task triggered by `scripts/integtest.sh` that can send those parameters into the test cases?

**Comment by DarshitChanpura** [comment]
> Have we considered replacing `OpenSearchRestTestCase` and creating a task triggered by `scripts/integtest.sh` that can send those parameters into the test cases?

That is how these tests are configured currently [1][2]

[1] https://github.com/DarshitChanpura/security/blob/main/src/test/java/org/opensearch/security/sanity/tests/SecurityRestTestCase.java
[2] https://github.com/DarshitChanpura/security/blob/main/README.md#test-and-build

**Comment by shikharj05** [comment]
Closing as 1.x is no longer maintained.
